content/en/event-driven_architecture.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Event-Driven Architecture
+status: Feedback Appreciated
+category: concept
+---
+
+## What it is
+It is a software architecture that promotes the creation, processing, and consumption of events. An event can be seen as any change of state or other significant occurance in hardware or software. This architecture creates the structure in which events can be properly routed from their source to the desired receivers.
```

Comment:
If 'occurance' is typo of 'occurrence', then please fix it. 😊

Reply:
That file is outdated, I updated a recent one